service/src/test/java/bio/terra/tanagra/service/accesscontrol/AouWorkbenchAccessControlTest.java

```diff
@@ -39,31 +41,14 @@ protected void cleanup() {
   @Test
   void underlay() {
     // isAuthorized, getPermissions, listAllPermissions, listAuthorizedResources
-    ResourceId cmsSynpufId = ResourceId.forUnderlay(CMS_SYNPUF);
     ResourceId aouSyntheticId = ResourceId.forUnderlay(AOU_SYNTHETIC);
-    ResourceId sddId = ResourceId.forUnderlay(SDD);
-    assertHasPermissions(USER_1, cmsSynpufId);
-    assertHasPermissions(USER_1, aouSyntheticId);
-    assertHasPermissions(USER_1, sddId);
-    assertHasPermissions(USER_2, cmsSynpufId);
-    assertHasPermissions(USER_2, aouSyntheticId);
-    assertHasPermissions(USER_2, sddId);
-    assertHasPermissions(USER_3, cmsSynpufId);
-    assertHasPermissions(USER_3, aouSyntheticId);
-    assertHasPermissions(USER_3, sddId);
-    assertHasPermissions(USER_4, cmsSynpufId);
-    assertHasPermissions(USER_4, aouSyntheticId);
-    assertHasPermissions(USER_4, sddId);
-
+    assertUnderlayPermissions(USER_1, aouSyntheticId);
+    assertUnderlayPermissions(USER_2, aouSyntheticId);
+    assertUnderlayPermissions(USER_4, aouSyntheticId);
     // service.list
-    assertServiceListWithReadPermission(
-        USER_1, ResourceType.UNDERLAY, null, true, cmsSynpufId, aouSyntheticId, sddId);
-    assertServiceListWithReadPermission(
-        USER_2, ResourceType.UNDERLAY, null, true, cmsSynpufId, aouSyntheticId, sddId);
-    assertServiceListWithReadPermission(
-        USER_3, ResourceType.UNDERLAY, null, true, cmsSynpufId, aouSyntheticId, sddId);
-    assertServiceListWithReadPermission(
-        USER_4, ResourceType.UNDERLAY, null, true, cmsSynpufId, aouSyntheticId, sddId);
+    assertResourceTypeNoListPermissions(USER_1, ResourceType.UNDERLAY);
```

Comment:
Is it possible to reuse an existing method here instead? e.g. `assertServiceListWithReadPermission(USER_1, ResourceType.UNDERLAY, /*parent*/ null, /*isAllResources*/ false)`

Reply:
yes that worked for underlays